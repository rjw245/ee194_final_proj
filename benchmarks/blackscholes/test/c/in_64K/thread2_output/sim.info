{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread2_output',
             '-n',
             '8',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../parsec-3.0/pkgs/apps/blackscholes/inst/amd64-linux.gcc-pthreads/bin/blackscholes',
             '2',
             '../../../inputs/in_64K.txt',
             'thread2_output/output.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (12120.370423,
            63.721312,
            135108,
            0,
            0,
            0,
            220922,
            0,
            0,
            0,
            14880,
            0,
            0,
            0,
            11435867,
            9995857),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=8 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/c/in_64K/thread2_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../parsec-3.0/pkgs/apps/blackscholes/inst/amd64-linux.gcc-pthreads/bin/blackscholes 2 ../../../inputs/in_64K.txt thread2_output/output.txt'],
 't_elapsed': 12328.598177909851,
 't_start': 1461613856.403299,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 811741184L}
