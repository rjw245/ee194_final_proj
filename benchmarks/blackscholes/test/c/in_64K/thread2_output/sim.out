                                     | Core 0      | Core 1      | Core 2      | Core 3      | Core 4      | Core 5      | Core 6      | Core 7     
  Instructions                       |   745611098 |  2372642082 |  2372644006 |           0 |           0 |           0 |           0 |           0
  Cycles                             |  2097648868 |  2097648868 |  2097648868 |  2097648868 |  2097648868 |  2097648868 |  2097648868 |  2097648868
  IPC                                |        0.36 |        1.13 |        1.13 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  Time (ns)                          |   788589800 |   788589800 |   788589800 |   788589800 |   788589800 |   788589800 |   788589800 |   788589800
  Idle time (ns)                     |   617736356 |   176048321 |   170856994 |   788589800 |   788589800 |   788589800 |   788589800 |   788589800
  Idle time (%)                      |       78.3% |       22.3% |       21.7% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0%
Branch predictor stats               |             |             |             |             |             |             |             |            
  num correct                        |   118143322 |   139738552 |   139462045 |           0 |           0 |           0 |           0 |           0
  num incorrect                      |     6830872 |     3891297 |     4167534 |           0 |           0 |           0 |           0 |           0
  misprediction rate                 |       5.47% |       2.71% |       2.90% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
  mpki                               |        9.16 |        1.64 |        1.76 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
TLB Summary                          |             |             |             |             |             |             |             |            
  I-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |   122006464 |   267910564 |   267910174 |           0 |           0 |           0 |           0 |           0
    num misses                       |         105 |          23 |          23 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |   255542040 |   760712194 |   760711781 |           0 |           0 |           0 |           0 |           0
    num misses                       |      334790 |    12058282 |    11909455 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.13% |       1.59% |       1.57% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.45 |        5.08 |        5.02 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  L2 TLB                             |             |             |             |             |             |             |             |            
    num accesses                     |      334895 |    12058305 |    11909478 |           0 |           0 |           0 |           0 |           0
    num misses                       |        1931 |        8002 |       14249 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.58% |       0.07% |       0.12% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.00 |        0.00 |        0.01 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
Cache Summary                        |             |             |             |             |             |             |             |            
  Cache L1-I                         |             |             |             |             |             |             |             |            
    num cache accesses               |   122006464 |   267910564 |   267910174 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |      250800 |       17191 |       17148 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.21% |       0.01% |       0.01% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |        0.34 |        0.01 |        0.01 |         inf |         inf |         inf |         inf |         inf
  Cache L1-D                         |             |             |             |             |             |             |             |            
    num cache accesses               |   255673224 |   760712202 |   760711787 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |      194099 |     2225998 |     2216597 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.08% |       0.29% |       0.29% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |        0.26 |        0.94 |        0.93 |         inf |         inf |         inf |         inf |         inf
  Cache L2                           |             |             |             |             |             |             |             |            
    num cache accesses               |      444962 |     2243192 |     2233747 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |      113413 |     1462544 |     1462817 |           0 |           0 |           0 |           0 |           0
    miss rate                        |      25.49% |      65.20% |      65.49% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |        0.15 |        0.62 |        0.62 |         inf |         inf |         inf |         inf |         inf
  Cache L3                           |             |             |             |             |             |             |             |            
    num cache accesses               |      113484 |     1462547 |     1462819 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |       67821 |        2302 |        2151 |           0 |           0 |           0 |           0 |           0
    miss rate                        |      59.76% |       0.16% |       0.15% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |        0.09 |        0.00 |        0.00 |         inf |         inf |         inf |         inf |         inf
DRAM summary                         |             |             |             |             |             |             |             |            
  num dram accesses                  |       36136 |           0 |           0 |           0 |       36137 |           0 |           0 |           0
  average dram access latency (ns)   |       60.79 |         inf |         inf |         inf |       60.76 |         inf |         inf |         inf
  average dram queueing delay        |        7.36 |        0.00 |        0.00 |        0.00 |        7.34 |        0.00 |        0.00 |        0.00
  average dram bandwidth utilization |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
Coherency Traffic                    |             |             |             |             |             |             |             |            
  num loads from dram                |        8061 |        2166 |        2069 |           0 |           0 |           0 |           0 |           0
  num loads from dram cache          |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from remote cache        |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
